aws-lc-fips-sys/include/rust_wrapper.h

```diff
@@ -6,26 +6,29 @@
 #ifndef OPENSSL_HEADER_RUST_WRAPPER_H
 #define OPENSSL_HEADER_RUST_WRAPPER_H
 
-#ifdef _WIN32
-#define BORINGSSL_SHARED_LIBRARY
-#define AWS_LC_FIPS_SYS_EXPORT __declspec(dllexport)
-#else
-#define AWS_LC_FIPS_SYS_EXPORT __attribute__((visibility("default")))
-#endif
-
 #include <openssl/err.h>
 
+#define AWS_LC_WRAPPER_ADD_PREFIX(a, b) AWS_LC_WRAPPER_ADD_PREFIX_INNER(a, b)
+#define AWS_LC_WRAPPER_ADD_PREFIX_INNER(a, b) a ## _ ## b
```

Comment:
Is there a particular reason why we can't just use
```
#define AWS_LC_WRAPPER_ADD_PREFIX(a, b) a ## _ ## b
```

Reply:
The same is done for `BORINGSSL_ADD_PREFIX` -- it allows for [macro expansion on the arguments](https://github.com/aws/aws-lc/blob/4cd6d216e6f13c1a3528dbbf68c2cc46f593ef44/util/make_prefix_headers.go#L109). But, in our limited case, this indirection is likely not necessary.